src/main/resources/assets/opencomputers/lang/en_us.json

```
@@ -31,53 +31,53 @@
     "item.opencomputers.abstractbuscard": "Abstract Bus Card",
     "item.opencomputers.grog": "Grog",
-    "item.opencomputers.alu": "Arithmetic Logic Unit (ALU)",
+    "item.opencomputers.arithmeticlogicunit": "Arithmetic Logic Unit (ALU)",
     "item.opencomputers.analyzer": "Analyzer",
-    "item.opencomputers.apu0": "§6Accelerated Processing Unit (APU) (Tier 2)",
-    "item.opencomputers.apu1": "Accelerated Processing Unit (APU) (Tier 3)",
-    "item.opencomputers.apu2": "Accelerated Processing Unit (APU) (Creative)",
+    "item.opencomputers.apu1": "§eAccelerated Processing Unit (APU) (Tier 2)",
+    "item.opencomputers.apu2": "§bAccelerated Processing Unit (APU) (Tier 3)",
+    "item.opencomputers.apucreative": "§5Accelerated Processing Unit (APU) (Creative)",
     "item.opencomputers.arrowkeys": "Arrow Keys",
     "item.opencomputers.buttongroup": "Button Group",
-    "item.opencomputers.cardbase": "Card Base",
+    "item.opencomputers.card": "Card Base",
     "item.opencomputers.chamelium": "Chamelium",
     "item.opencomputers.circuitboard": "Circuit Board",
     "item.opencomputers.componentbus0": "Component Bus (Tier 1)",
-    "item.opencomputers.componentbus1": "§6Component Bus (Tier 2)",
-    "item.opencomputers.componentbus2": "Component Bus (Tier 3)",
-    "item.opencomputers.componentbus3": "Component Bus (Creative)",
+    "item.opencomputers.componentbus1": "§eComponent Bus (Tier 2)",
+    "item.opencomputers.componentbus2": "§bComponent Bus (Tier 3)",
+    "item.opencomputers.componentbus3": "§5Component Bus (Creative)",
     "item.opencomputers.controlunit": "Control Unit (CU)",
     "item.opencomputers.cpu1": "Central Processing Unit (CPU) (Tier 1)",
-    "item.opencomputers.cpu2": "§6Central Processing Unit (CPU) (Tier 2)",
-    "item.opencomputers.cpu3": "Central Processing Unit (CPU) (Tier 3)",
+    "item.opencomputers.cpu2": "§eCentral Processing Unit (CPU) (Tier 2)",
+    "item.opencomputers.cpu3": "§bCentral Processing Unit (CPU) (Tier 3)",
     "item.opencomputers.cuttingwire": "Cutting Wire",
     "item.opencomputers.datacard1": "Data Card (Tier 1)",
-    "item.opencomputers.datacard2": "§6Data Card (Tier 2)",
-    "item.opencomputers.datacard3": "Data Card (Tier 3)",
+    "item.opencomputers.datacard2": "§eData Card (Tier 2)",
+    "item.opencomputers.datacard3": "§bData Card (Tier 3)",
     "item.opencomputers.debugcard": "Debug Card",
     "item.opencomputers.debugger": "Network Debugger",
     "item.opencomputers.diamondchip": "Diamond Chip",
     "item.opencomputers.disk": "Disk Platter",
     "item.opencomputers.diskdrivemountable": "Disk Drive",
     "item.opencomputers.drone": "Drone",
     "item.opencomputers.dronecase0": "Drone Case (Tier 1)",
-    "item.opencomputers.dronecase1": "§6Drone Case (Tier 2)",
-    "item.opencomputers.dronecase3": "Drone Case (Creative)",
+    "item.opencomputers.dronecase1": "§eDrone Case (Tier 2)",
+    "item.opencomputers.dronecase3": "§5Drone Case (Creative)",,
```

```suggestion
    "item.opencomputers.dronecase3": "§5Drone Case (Creative)",
```

There are some duplicated commas :)